sympy/stats/crv.py

```diff
@@ -336,12 +336,17 @@ def where(self, condition):
         return SingleContinuousDomain(rv.symbol, interval)
 
     def conditional_space(self, condition, normalize=True, **kwargs):
-
         condition = condition.xreplace(dict((rv, rv.symbol) for rv in self.values))
-
         domain = ConditionalContinuousDomain(self.domain, condition)
         if normalize:
-            pdf = self.pdf / domain.integrate(self.pdf, **kwargs)
+            # create a clone of the variable to
+            # make sure that variables in nested integrals are different
+            # from the variables outside the integral
+            # this makes sure that they are evaluated separately
+            # and in the correct order
+            replacement  = {rv: Dummy(str(rv)) for rv in self.symbols}
+            pdf = self.pdf / domain.integrate(self.pdf, **kwargs)    \
+                                             .xreplace(replacement)
```

Comment:
I think we should try to avoid white space next to '.' although that is lexically allowed. Instead, two separate statements could be used:

    norm = domain.integrate(self.pdf, **kwargs)
    pdf = self.pdf / norm.xreplace(replacement)

Otherwise this looks good.

Reply:
Will do!